src/borg/archive.py

```diff
@@ -793,7 +792,7 @@ def restore_attrs(self, path, item, symlink=False, fd=None):
                         # EACCES: permission denied to set this specific xattr (this may happen related to security.* keys)
                         # EPERM: operation not permitted
                         err_str = os.strerror(e.errno)
-                    logger.warning(msg_format % err_str)
+                    logger.warning('%s: when setting extended attribute %s: %s' % (path, k.decode(), err_str))
```

Comment:
thanks for discovering and fixing this.

the above code is a bit better, but still problematic, because the first arg of `logger.warning` is interpreted as a format string, see:

https://docs.python.org/3/library/logging.html#logging.Logger.debug

Reply:
so guess it needs to be:
```
logger.warning(format, path, k.decode(), err_str)
```

Reply:
and also guess we need to look for other such places in the code.